gamemode/modules/drowning/sv_drowning.lua

```diff
@@ -0,0 +1,88 @@
+local firstSubmergedTime = {}
+local lastDrownTick = {}
```

Comment:
could you just store  this stuff on the player entity itsself instead of a table, you wouldnt need to worry about cleaning any values up on player disconnect